src/metrics.rs

```diff
@@ -18,6 +18,15 @@ use prometheus::core::Collector;
 pub use prometheus::Result;
 use prometheus::{HistogramOpts, Opts, Registry, DEFAULT_BUCKETS};
 
+/// "event" is used as a label for Metrics that can apply to both Filter
+/// `read` and `write` executions.
+pub const EVENT_LABEL: &str = "event";
```

Comment:
Is there a way to do this with a type system? I couldn't think of one.

Reply:
You could have a trait with associated constants, though I'm not sure how generally applicable it might be.

```rust
pub trait FilterMetricLabels {
    const LABEL: &'static str;
    const READ: &'static str = "read";
    const WRITE: &'static str = "write";
}

impl FilterMetricsLabel for firewall::Metrics {
    const LABEL: &'static str = "event";
}
```